From b108f741b6ebd1e68a0392a72ba8e35537689ba0 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@hadrons.org>
Date: Tue, 19 Apr 2022 15:45:39 +0200
Subject: [PATCH libaio 16/16] harness: Disable failing test 23
Origin: vendor
Forwarded: not-needed

This new test fails on several architectures. But it is a new test
intended to check for a Linux kernel regression, so we are not worse
off than were before. Add it to the EXTRACASES variables, which has
currently the side effect of disabling it, as it is not handled
explicitly in the make target responsible for that variable.

Sent-Upstream: not-relevant
Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 harness/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/harness/Makefile b/harness/Makefile
index ae4e45e..ff0a041 100644
--- a/harness/Makefile
+++ b/harness/Makefile
@@ -1,6 +1,6 @@
 # foo.
 TEST_SRCS:=$(shell find cases/ -name \*.t | sort -n -t/ -k2)
-EXTRAPROGS:=cases/8.p cases/10.p
+EXTRAPROGS:=cases/8.p cases/10.p cases/23.p
 PARTPROGS:=$(filter-out $(EXTRAPROGS), $(patsubst %.t,%.p,$(TEST_SRCS)))
 PROGS:=$(PARTPROGS) $(EXTRAPROGS)
 HARNESS_SRCS:=main.c
-- 
2.40.1

